---
ElasticFusionOutput elastic_fusion_output